### PR TITLE
Add loadPaymentNotes to saveMemberPayment persistence wrapper

### PR DESCRIPTION
**Pull request by amberwebb**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # TeamSnap JavaScript SDK CHANGELOG
 
+### March 2, 2016 // Version 1.12.0
```
Comment (matthopson) [inline CHANGELOG.md]: Should be a patch

File: src/persistence.coffee [hunk 2]
```diff
@@ -498,9 +498,11 @@ modifySDK = (sdk) ->
       saveMemberPayment.call(this, memberPayment).then((result) ->
         memberId = result.memberId
         teamFeeId = result.teamFeeId
+        teamId = result.teamId
         promises.when(
           sdk.loadMemberBalances(memberId: memberId)
           sdk.loadTeamFees(id: teamFeeId)
+          sdk.loadPaymentNotes(teamId: teamId)
```
Comment (matthopson) [inline src/persistence.coffee]: We should probably not load the whole team's notes - at the very least we could do it on member, but we should be able to use the teamFee as well and be more pacific.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
...
+- Reloads `loadPaymentNotes` in `saveMemberPayment` method in persistence wrapper.
```
Comment (matthopson) [inline CHANGELOG.md]: "Reloads `paymentNotes`..."

File: src/persistence.coffee [hunk 4]
```diff
@@ -501,6 +501,7 @@ modifySDK = (sdk) ->
         promises.when(
           sdk.loadMemberBalances(memberId: memberId)
           sdk.loadTeamFees(id: teamFeeId)
+          sdk.loadPaymentNotes(teamFeeId: teamFeeId)
```
Comment (matthopson) [inline src/persistence.coffee]: Was looking at `v3` - it seems we can query on `memberPaymentId`
https://github.com/teamsnap/apiv3/blob/master/app/payment_notes/payment_note_serializer.rb#L34

**Comment by amberwebb** [comment]
@matthopson Please review again.

**Comment by matthopson** [comment]
:+1:

**Pull request merged**